src/Compiler/Microsoft.AspNetCore.Razor.Language/legacyTest/TestFiles/ParserTests/CSharpBlockTest/TreatsAtSignsAfterFirstPairAsPartOfCSharpStatement.diag.txt

```diff
@@ -0,0 +1,5 @@
+﻿(1,14): Error RZ1009: The "@" character must be followed by a ":", "(", or a C# identifier.  If you intended to switch to markup, use an HTML start tag, for example:
```

Comment:
Nit: we could change the message to specify this only applies in a code block.

Reply:
I'm going to avoid changing this message for now.